### PR TITLE
FAT-6955: Initial implementation

### PR DESCRIPTION
**Pull request by zentestuken**
FAT ticket - https://issues.folio.org/browse/FAT-6955

Test case - https://foliotest.testrail.io/index.php?/cases/view/359596

Allure report - https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/3517/allure/

### CODE REVIEW

File: cypress/e2e/inventory/search/browse-contributors-linked.cy.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+import Logs from '../../../support/fragments/data_import/logs/logs';
+import InventoryInstance from '../../../support/fragments/inventory/inventoryInstance';
+import InventoryInstances from '../../../support/fragments/inventory/inventoryInstances';
+import InventorySearchAndFilter from '../../../support/fragments/inventory/inventorySearchAndFilter';
+import BrowseContributors from '../../../support/fragments/inventory/search/browseContributors';
+import QuickMarcEditor from '../../../support/fragments/quickMarcEditor';
+import MarcAuthority from '../../../support/fragments/marcAuthority/marcAuthority';
+import MarcAuthorities from '../../../support/fragments/marcAuthority/marcAuthorities';
+
+describe('Browse Contributors in Inventory (linked records)', () => {
```
Comment (IhorBohdan) [inline cypress/e2e/inventory/search/browse-contributors-linked.cy.js:18]: Maybe here better named like in testrail:
Inventory:Contributors Browse?

File: cypress/e2e/inventory/search/browse-contributors-linked.cy.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+          Logs.openFileDetails(marcFile.fileName);
+          for (let i = 0; i < marcFile.numOfRecords; i++) {
+            Logs.getCreatedItemsID(i).then(link => {
+              createdRecordIDs.push(link.split('/')[5]);
+            });
+          }
+        });
+      });
+
+      cy.loginAsAdmin({ path: TopMenu.inventoryPath, waiter: InventoryInstances.waitContentLoading }).then(() => {
```
Comment (IhorBohdan) [inline cypress/e2e/inventory/search/browse-contributors-linked.cy.js:82]: You are already login as Admin

File: cypress/e2e/inventory/search/browse-contributors-linked.cy.js (hunk 1 above)
Comment (zentestuken) [reply cypress/e2e/inventory/search/browse-contributors-linked.cy.js]: Updated block name to "Inventory: Contributors Browse"

File: cypress/e2e/inventory/search/browse-contributors-linked.cy.js (hunk 2 above)
Comment (zentestuken) [reply cypress/e2e/inventory/search/browse-contributors-linked.cy.js]: Replaced `cy.loginAsAdmin()` with `cy.visit()` and `InventoryInstances.waitContentLoading()`

File: cypress/support/fragments/inventory/inventoryInstance.js [hunk 3]
```diff
@@ -401,6 +401,15 @@ export default {
     cy.expect(authoritySearchResults.exists());
   },
 
+  searchResultsWithOption(option, value) {
+    cy.do(selectField.choose(including(option)));
```
[the remark this answers is not in the document]
Comment (zentestuken) [reply cypress/support/fragments/inventory/inventoryInstance.js]: Moved consequent actions to an array within `cy.do()`

File: cypress/support/fragments/inventory/inventoryInstance.js [hunk 4]
```diff
@@ -401,6 +401,17 @@ export default {
     cy.expect(authoritySearchResults.exists());
   },
 
+  searchResultsWithOption(option, value) {
+    cy.do([
+      selectField.choose(including(option)),
+      searchInput.fillIn(value)
+    ]);
+    cy.expect(searchInput.has({ value }));
```
Comment (IhorBohdan) [inline cypress/support/fragments/inventory/inventoryInstance.js:409]: Also move cy.expect to array

File: cypress/e2e/inventory/search/browse-contributors-linked.cy.js [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+          cy.wait(1500);
+        });
+        QuickMarcEditor.pressSaveAndClose();
+        InventoryInstance.waitLoading();
+      });
+    });
+  });
+
+  beforeEach('Login to the application', () => {
+    cy.login(testData.userProperties.username, testData.userProperties.password, { path: TopMenu.inventoryPath, waiter: InventoryInstances.waitContentLoading });
```
Comment (Sherzod-Kenjaev) [inline cypress/e2e/inventory/search/browse-contributors-linked.cy.js:106]: If we have just one test in the file we can move cy.login() to "before" hook.

Comment (zentestuken) [reply cypress/e2e/inventory/search/browse-contributors-linked.cy.js]: Moved as proposed

File: cypress/support/fragments/inventory/inventoryInstance.js (hunk 4 above)
Comment (zentestuken) [reply cypress/support/fragments/inventory/inventoryInstance.js]: Moved two conseсutive `expect()` checks into one array

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**